### ISSUE TITLE
Unifier failing due to missing junction files

### ISSUE THREAD

**Issue by beeCwright** [langmead-lab/monorail-external#12]
I'm running the recount-unifer on the provided sample data but the pipeline is failing with the following error in the log:

![image](https://user-images.githubusercontent.com/19931750/148824929-52ca3302-ea28-43b3-89a1-8a8ed7e9695b.png)

The pump ran correctly

Images:
quay.io/broadsword/recount-unify   1.0.9
quay.io/benlangmead/recount-rs5    1.0.6

ids.tsv
SRP020237       SRR390728       831056

ids.input
SRP020237       SRR390728

The step it is failing on takes the following file as input. This file exists, but is empty.
junction_counts_per_study/SRP020237.unique.sj.merged.motifs.annotated 

Any suggestions on what could be causing this? I'm using the example fastq data files.

**Comment by ChristopherWilks** [comment]
hi @beeCwright,
Thanks for your patience.  I finally got a chance to run this myself (again) and realized that there'd been some major changes to the unifier image since I last tested the main README.

Anyway, the tl;dr version is that the test case was *too* small (only ~125 paired reads) to generate any splice junctions causing the 2nd part of the unifier where junctions are aggregated to fail since there are 0 junctions in the input to it.

I've updated the README to use a larger sample which I've just now tested with the versions of the images you're using and it worked as it has a number of splice junctions.

I suggest you update to the new sample, re-run pump and unifier and see if that works:

http://snaptron.cs.jhu.edu/data/temp/SRR390728_1.fastq.gz
http://snaptron.cs.jhu.edu/data/temp/SRR390728_2.fastq.gz
